src/box/applier.cc

```diff
@@ -170,13 +170,14 @@ applier_check_sync(struct applier *applier)
  * but doesn't start it.
  */
 static struct fiber *
-applier_fiber_new(struct applier *applier, const char *name, fiber_func func)
+applier_fiber_new(struct applier *applier, const char *name, fiber_func func,
+		  bool is_joinable)
```

Comment:
IMO this patch should be split into 2 commits: one adding is_joinable argument to applier_fiber_new, second changing fiber_system_new to applier_fiber_new for ballot_watcher in applier_connect.

Reply:
I thought the change was too tiny to split it into 2 commits, but I don't mind. Done.